lib/board.rb

```diff
@@ -1,14 +1,14 @@
 class Board
-  def initialize
-    @board = [
-      [nil, nil, nil],
-      [nil, nil, nil],
-      [nil, nil, nil]
-    ]
+  def initialize(width:)
+    @board = generate_empty_board(width: width)
```

Comment:
Were we talking about class Board having instance variable called board as well being weird? I don't remember, but think that we talked about it. Maybe rename it? something like spots? cells? squares? I don't know what will make more sense.

Reply:
I think we ended up creating `state` as a way to access `@board` from outside. I will rename `@board` to `@state`